### ISSUE TITLE
Publish as installable MultiMC pack

### ISSUE THREAD

**Issue by comp500** [packwiz/packwiz-example-pack#1]
Demonstrate how to create a MultiMC pack for distribution with packwiz-installer; likely hosted on Netlify.

**Comment by ChipWolf** [comment]
Honestly, there could be an endpoint that you reference a `pack.toml` as a query attribute & it could serve a MultiMC zip ready to go [with some caching]; i.e. `packwiz.dev/multimc?pack=example.com%2Fpack.toml`

This could be further enhanced with a URL shortener if someone wanted to manage the stateful overhead; i.e. `packwiz.dev/slug`

**Comment by comp500** [comment]
Yeah I was thinking of doing this clientside in https://github.com/packwiz/packwiz/issues/76. Though now that I'm working towards packwiz-installer having local configuration, I might just publish a universal "packwiz-installer MultiMC pack" and have the `pack.toml` URL be configured with a user interface on first launch.
